src/opm/output/eclipse/AggregateGroupData.cpp

```diff
@@ -585,7 +521,7 @@ captureDeclaredGroupData(const Opm::Schedule&                 sched,
     curGroups.resize(ngmaxz(inteHead));
     //
     //initialize curgroups
-    for (auto* cg : curGroups) cg = nullptr;
+    for (auto*& cg : curGroups) cg = nullptr;
 
```

Comment:
Just put the `nullptr` initaliation into the `resize()` call.

Reply:
done.

Reply:
Come to think of it, it might just be better to fully construct `curGroups` in one go as
```
std::vector<const Group*> curGroups(ngmaxz(inteHead), nullptr);
```

Reply:
done.